content/_help/verify-your-identity/accepted-state-issued-identification._en.md

```diff
@@ -1,33 +1,29 @@
 ---
 layout: help
-title: Accepted state-issued identification
+title: Accepted identification documents
 category: verify-your-identity
 permalink: /help/verify-your-identity/accepted-state-issued-identification/
```

Comment:
Since we're updating the title, do we want to update the URL as well to match?

(And if we do, we should also ensure that we have a redirect in place from the old URL to the new with `redirect_from` below)

Reply:
@aduth , discussed with @kellular , my impression is that we keep those intact? Is it correct @kellular ?

Reply:
I think we generally keep the URL matching the title of the page, so if we're updating the title, I'd expect we'd also update the URL.

i.e. `/help/verify-your-identity/accepted-identification-documents/`

Reply:
@aduth, I was also thinking,  should we rename the markdown files also,  what's the general practice here?

Reply:
Yeah I think that makes sense. Good call

Reply:
@aduth , changes made.